src/__tests__/views/articleDetailView/__test__/__snapshots__/articleDetailView.test.js.snap

```diff
@@ -63,7 +63,7 @@ ShallowWrapper {
     "nodeType": "function",
     "props": Object {
       "children": Array [
-        <Connect(Navbar) />,
+        <Navbar />,
```

Comment:
why did you remove the connect from Navbar component ?

Reply:
It's because the navbar does not need to connect to redux.